script/test.bash

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+yarn test || exit 111
```

Comment:
why run `yarn test` from here instead of running this from `yarn test`?

Reply:
It felt a little weird to run shell scripts from `yarn test` but we already run `build-simulations.bash` from it so I can move it there